mu/logic.py

```diff
@@ -413,7 +463,12 @@ def flash(self):
             logger.debug('Flashing to device.')
             hex_file = os.path.join(path_to_microbit, 'micropython.hex')
             uflash.save_hex(micropython_hex, hex_file)
+            rt_hex_path = get_runtime_hex_path()
             message = 'Flashing "{}" onto the micro:bit.'.format(tab.label)
+            if (rt_hex_path != "" and os.path.exists(rt_hex_path)):
```

Comment:
Rather pointless checking if the path exists now (things would have already broken)